index.html

```diff
@@ -408,6 +408,28 @@ <h1>Introduction</h1>
       </p>
     </section>
 
+    <section>
+      <h1>Lifecycle of a WoT device</h1>
+
+      <figure id="wot-lifecycle">
+           <img src="images/WoT_lifecycle_diagram.png"
+                width="100%"

```

Comment:
About the picture, I like that it has the lifecycle states in one line (and I see the improvement from the previous version in this regard).

Perhaps "D. Re-installation required" could be moved out and on the top around the provisioning state machine, but I am not sure it will actually look better. This should be fine.

The states and options are pretty clear (I formulated a few questions, then I figured out the picture is ok - good work :).